app/views/logs/log.njk

```diff
@@ -20,7 +20,7 @@
       <h2 class="govuk-heading-s govuk-!-margin-bottom-2">This log is incomplete</h2>
       <p class="govuk-body govuk-!-margin-bottom-7">
         You have completed 0 of {{ data.sections | length }} sections.<br>
-        Skip to next incomplete section: <a href="{{ nextSection(log.id).href }}">{{ nextSection(log.id).text }}</a>
+        Skip to next incomplete section: <a href="#item-{{ nextSection(log.id).text | lower | replace(" ", "-") }}">{{ nextSection(log.id).text }}</a>
```

Comment:
Can you not do this?

```suggestion
        Skip to next incomplete section: <a href="#item-{{ nextSection(log.id).id }}">{{ nextSection(log.id).text }}</a>
```

Reply:
What do you mean by "not do this"?

Reply:
Doesn’t each section have a kebab-cased `id` value already, meaning you don’t need to convert the name to kebab case?

Reply:
not in the `nextSection` obj - but good call about separating concerns. I made a change.